lib/logstash/outputs/influxdb.rb

```diff
@@ -20,7 +20,10 @@ class LogStash::Outputs::InfluxDB < LogStash::Outputs::Base
   config_name "influxdb"
 
   # The database to write
-  config :db, :validate => :string, :default => "stats"
+  config :db, :validate => :string, :default => "statistics"
```

Comment:
(caveat, I have no experience with influxdb)

This changes the default db name which I would guess causes anyone who upgrades to this plugin to leave their old dataset behind - is this right? This seems a change that breaks backwards compatibility.